### ISSUE TITLE
KIS, MMC, RMC, SML, TIU, and WRH holdings are not displaying

### ISSUE THREAD

**Issue by patrickzurek** [CARLI/vufind#26]
JIRA issue created by: Jessica Gibson (gibsonjc)
Originally opened: 2016-01-11 03:55 PM

No holdings for KIS, MMC, RMC, SML, TIU, and WRH are displaying in vufind2-test.carli.illinois.edu/all/ or vufind2-test.carli.illinois.edu/vf-xxx/ 

The "Holdings" tab is blank.

Is this because they were standalone Voyager instances before joining I-Share and have something different in their database configurations? For example, Gordon reports that the dbkeys for these six are not in the same form that the other I-Share libraries have.

**Comment by patrickzurek** [comment]
JIRA User: Chris Delis (cedelis)
JIRA Timestamp: 2016-01-12 12:05 PM

I noticed that on the vufind UI (vufind.carli.illinois.edu) in vf-xxx/config/vufind/XCNCIP.ini, that the 003s were incorrect or missing for all these libraries:

KISHdb -> KISdb
Cdb -> RMCdb
added SMLdb 
added TIUdb
added WRHdb

After fixing this, I noticed that SML was still not working. I had to fix a config error on the ncip server side, file /etc/tomcat7/Catalina/localhost/voyager.xml 

For some reason SML's config was missing (and instead a setting for CDB???). I changed CDB to SML and now it works.
